### PR TITLE
Add a post-submit job for knative-sandbox/net-istio to run with latest Istio version.

### PR DESCRIPTION
**Pull request by tcnghia**
**What this PR does, why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes to reviewers**:

**User-visible changes in this PR**:

### CODE REVIEW

**Comment by tcnghia** [comment]
cc @JRBANCEL @ZhiminXiang

File: config/prod/prow/jobs/config.yaml [hunk 1]
```diff
@@ -8819,6 +8819,39 @@ periodics:
...
       secret:
         secretName: nightly-account
+- cron: "49 */2 * * *"
+  name: ci-knative-sandbox-net-certmanager-latest
+  agent: kubernetes
+  decorate: true
+  cluster: "build-knative"
+  extra_refs:
+  - org: knative-sandbox
+    repo: net-certmanager
```
Comment (ZhiminXiang) [inline config/prod/prow/jobs/config.yaml:8829]: Just want to double check the repo is net-certmanager not net-istio? (as in the description of this PR, it mentions net-istio)

Comment (chizhg) [reply config/prod/prow/jobs/config.yaml:8829]: Ah it looks it was added at the wrong place... @tcnghia could you confirm please?

**Pull request merged**